kong/db/migrations/core/005_120_to_130.lua

```diff
@@ -104,6 +104,8 @@ return {
         PRIMARY KEY (partition, id)
       );
 
+      CREATE INDEX IF NOT EXISTS ca_certificates_cert_idx ON ca_certificates(cert);
```

Comment:
Please remove this:

The `UNIQUE` constraint is implemented in `Postgres` as unique index.

Reply:
curious @bungle : so if you add a UNIQUE constraint, then Postgres automatically adds an index already on that field?

Reply:
@Tieske that's right. But in this case the index was for Cassandra.

Reply:
@Tieske see also: https://www.postgresql.org/docs/12/indexes-unique.html
> There's no need to manually create indexes on unique columns; doing so would just duplicate the automatically-created index.

Or https://www.postgresql.org/docs/9.4/indexes-unique.html:
> Note: The preferred way to add a unique constraint to a table is ALTER TABLE ... ADD CONSTRAINT. The use of indexes to enforce unique constraints could be considered an implementation detail that should not be accessed directly. One should, however, be aware that there's no need to manually create indexes on unique columns; doing so would just duplicate the automatically-created index.